MediaAdminBundle/Resources/public/coffee/groupTree/FormMediaFolderCollectionView.coffee

```diff
@@ -0,0 +1,36 @@
+FormMediaFolderCollectionView = OrchestraView.extend(
```

Comment:
you use the syntax with class and also add namespace you have an example with view AjaxLoaderView